### DISCUSSION TITLE
v4.0.0 Upgrade

### DISCUSSION THREAD

**Discussion by followbl** [mohsinulhaq/react-popper-tooltip#110] [General]
Thanks for the upgrade work @mohsinulhaq went really smooth - only thing I would note - or might consider adding to your release notes is the css for positioning was a breaking change for me.

Again - thanks for the work 🎈 🎉 🥇

**Comment by mohsinulhaq** [comment]
@followbl just to clarify, have you copied our CSS file content to your own instead of importing it directly?

**Reply by followbl** [reply]
Yea - just because I wanted to adjust the tooltip css ever so slightly. Totally on me - just figured I'd flag it as it may help others with the upgrade
